sa3ed_app/www/found_person_create_detail/index.js

```diff
@@ -1,6 +1,23 @@
 var app = Vue.createApp({
     delimiters: ['[[', ']]'], // Change delimiters here
     data() {
+        frappe.call({
+            method:"sa3ed_app.api.WhitelistBypass.get_country_list",
+            callback: function(result) { 
+                message = result.message;
```

Comment:
Return `status_code` so you can verify that the API request was fulfilled.
Look to [ApiEndPoint.py](https://github.com/lavaloon-eg/sa3ed_app/blob/master/sa3ed_app/api/ApiEndPoint.py) and [FoundPersonEndPoints.py](https://github.com/lavaloon-eg/sa3ed_app/blob/33520d11731854093acc45a4bcd826efa7fc0bc7/sa3ed_app/api/FoundPersonEndPoints.py#L69)

Reply:
This issue should be solved by Commit 2c03058